packages/warn/src/index.ts

```diff
@@ -7,11 +7,11 @@
  * For example: stats warns when any tracks events aren't properly formatted (@see lib/analytics)
  */
 
-let warn;
+let warn: typeof console.warn;
 if ( process.env.NODE_ENV === 'production' || 'function' !== typeof console.warn ) {
-	warn = () => {};
+	warn = (): void => {};
```

Comment:
Putting some `/* do no logging at all */` comment inside the function body fixes the the `no-empty-function` Lint warning and is also useful.

Reply:
What Jarda said.